src/main/proto/ga4gh/sequence_annotation_service.proto

```diff
@@ -70,41 +70,48 @@ message SearchFeaturesRequest {
   // The annotation set to search within. Either `feature_set_id` or
   // `parent_id` must be non-empty.
   string feature_set_id = 1;
+  
+  // The name of this feature.
+  string name = 2;
+  
+  // The symbol for the gene that this feature occurs on. This field may be
+  // replaced with a more generic representation in a future version.
+  string gene_symbol = 3;
```

Comment:
Same problem here. This is just describing the field (which is already done). This should be documenting the _search_.